src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useCallback, useMemo, useState } from "react";
+import { Nav } from "react-bootstrap";
+import classNames from "classnames";
+
+import { Venue_v2 } from "types/venues";
+
+import { useVenueId } from "hooks/useVenueId";
+import { useConnectCurrentVenueNG } from "hooks/useConnectCurrentVenueNG";
+import { useUser } from "hooks/useUser";
+import { useRoles } from "hooks/useRoles";
+import { useIsAdminUser } from "hooks/roles";
+
+import AdvancedSettings from "pages/Admin/AdvancedSettings";
+import EntranceExperience from "pages/Admin/EntranceExperience";
+import VenueWizard from "pages/Admin/Venue/VenueWizard/VenueWizard";
+import { LoadingPage } from "components/molecules/LoadingPage";
+
+import "./AdminAdvancedSettings.scss";
+
+export enum AdminAdvancedTab {
+  basicInfo = "basic_info",
+  entranceExperience = "entrance_experience",
+  advancedMapSettings = "advanced_map_settings",
+}
+
+const adminAdvancedTabLabelMap: Readonly<Record<AdminAdvancedTab, String>> = {
+  [AdminAdvancedTab.basicInfo]: "Start",
+  [AdminAdvancedTab.entranceExperience]: "Entrance",
+  [AdminAdvancedTab.advancedMapSettings]: "Advanced",
+};
+
+const DEFAULT_TAB = AdminAdvancedTab.basicInfo;
+
+export const AdminAdvancedSettings: React.FC = () => {
+  const venueId = useVenueId();
+  const [selectedTab, setSelectedTab] = useState<AdminAdvancedTab>(DEFAULT_TAB);
+
+  const { userId } = useUser();
+  const { roles } = useRoles();
+  const { isAdminUser } = useIsAdminUser(userId);
+
+  const selectDefaultTab = useCallback(() => {
+    setSelectedTab(DEFAULT_TAB);
+  }, []);
+
+  const {
+    currentVenue: venue,
+    isCurrentVenueLoaded,
+  } = useConnectCurrentVenueNG(venueId);
+
+  const renderAdminAdvancedTabs = useMemo(() => {
+    return Object.entries(adminAdvancedTabLabelMap).map(([key, text]) => (
+      <Nav.Link
+        key={key}
+        className={classNames("AdminVenueView__tab", {
+          "AdminVenueView__tab--selected": selectedTab === key,
+        })}
+        eventKey={key}
+      >
+        {text}
+      </Nav.Link>
+    ));
+  }, [selectedTab]);
+
+  const selectTab = useCallback((tab: string) => {
+    setSelectedTab(tab as AdminAdvancedTab);
+  }, []);
```

Comment:
any reason to wrap this into a function? Looks like you can just use `setSelectedTab`

Reply:
We should avoid inline functions and use `useCallback` to avoid unnecessary re-renders. In this case I need to cast the parameter because it's a string.

Ex: 
```tsx
<Nav
  className="AdminAdvancedSettings__options"
  activeKey={selectedTab}
  onSelect={(value) => setSelectedTab(value as AdminAdvancedTab)}
>
```